venicegeo/eventkit-cloud
# fixed license checking

Licenses are validated against the job creator instead of the requestor which causes filtering to fail when a user with a valid license is viewing datapacks.

This verifies licenses against the requestor.  

To test, 
1. Add a license to a data source.  
2. Create a second user (User2). 
3. As User2, create a job for the data source with the license from step 1.  
4. Change the license (i.e. create a new one) for the license in step 1. 
5. As the default user (admin) try to view the datapacks.   Datapacks should be displayed properly.
